controllers/configurationpolicy_controller.go

```diff
@@ -1074,7 +1077,7 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 				convertPolicyStatusToString(&plc))
 		}
 
-		r.checkRelatedAndUpdate(plc, relatedObjects, oldRelated, statusUpdateNeeded)
+		r.checkRelatedAndUpdate(plc, relatedObjects, oldRelated, statusUpdateNeeded, false)
```

Comment:
I think this one should be `true`, this case could come up if they have a template with a range, but it changes to be empty. In this case, I think we could assume that it's working correctly, and they might want to remove the objects that used to be there.